Hearts And Minds

Big-Daddy-Grimm opened the pull request.

HuTcHGER commented:
Hi,
i've some problems/missunderstanding with H+M and i'm not sure if thats right here to write down:

1. Better understanding to open the garage for admins ?
2. it is possible to add static enemy units inside houses ?
3. it is possible to add flying enemy units , like patrols or paradrops ?

Maybe you can help me with that 
Thx

PS: Good work , i love to playing H+M for hours

1kuemmel1 committed:
adjust entry for format usage

Vdauphin committed:
Merge pull request #438 from 1kuemmel1/FIX-deploy_FOB_hint

FIX: deploy FOB hint, missing space

Vdauphin committed:
FIX: interaction menu for addon not following A3 standard

Vdauphin committed:
FIX: inherent

Vdauphin committed:
Remove lines

1kuemmel1 committed:
add missing basic value

Vdauphin committed:
Merge pull request #440 from Vdauphin/FIX-civi_intearation

FIX: Civilian H&M-interaction for mod

Vdauphin committed:
Merge pull request #442 from 1kuemmel1/FIX-param_rearm_default_selection

FIX: Missing basic value in rearm (param.hpp)

Vdauphin committed:
FIX: syntaxe error

Vdauphin committed:
Merge pull request #443 from Vdauphin/FIX-getout_error

FIX: Syntaxe error for setWaypointType

Vdauphin committed:
Merge pull request #428 from Vdauphin/Add-specific_icon_to_order

Add: Specific Icon to order action

Vdauphin committed:
Use task id for private

Vdauphin committed:
Merge remote-tracking branch 'refs/remotes/origin/master_daily' into ADD-change_task

Vdauphin committed:
FIX: Show hint for side

Vdauphin committed:
Use isEqualType

Vdauphin committed:
Compile "compile.sqf" file before JIP are executed

Extended_PreInit_EventHandlers compile the compile.sqf file before the
JIP remoteExec "btc_fnc_" is executed

Vdauphin committed:
Merge remote-tracking branch 'refs/remotes/origin/master_daily' into Add-rep_civ_car

# Conflicts:
#	=BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml

Vdauphin committed:
Decrease theshold

Vdauphin committed:
FIX: rtp error when eh are removed twice

Vdauphin committed:
Merge pull request #444 from Vdauphin/FIX-remove_eh

FIX: RTP errors when EH are removed twice

Vdauphin committed:
FIX: missing factions due to mod update

Vdauphin committed:
Merge pull request #445 from Vdauphin/FIX-missing_factions

FIX: Missing factions due to mods update

Vdauphin committed:
Merge pull request #430 from 1kuemmel1/Add-marker_handling_client

Add: Marker texts are now localized to client

1kuemmel1 committed:
changed init

1kuemmel1 committed:
fixed: no reaction to orders (civ: mod factions)

1kuemmel1 committed:
added fix from #440

1kuemmel1 committed:
Merge branch 'master_daily' into Remove-double_civ_actions

1kuemmel1 committed:
Merge branch 'master_daily' into Remove-double_civ_actions

Vdauphin committed:
FIX: no reaction to orders (civ: mod factions)

https://github.com/Vdauphin/HeartsAndMinds/pull/439/commits/3bdb933c410557ebb7c101a5e49496d7d500a4e2

Vdauphin committed:
Merge pull request #446 from Vdauphin/FIX-no_civ_reaction

FIX: No reaction to orders (civilian from mod factions)

1kuemmel1 committed:
use add_actions, cleanup

Vdauphin committed:
Add: 1.17.5

Vdauphin committed:
Merge remote-tracking branch 'refs/remotes/origin/master_stable' into master_daily

# Conflicts:
#	=BTC=co@30_Hearts_and_Minds.Altis/core/def/param.hpp
#	=BTC=co@30_Hearts_and_Minds.Altis/core/doc.sqf
#	=BTC=co@30_Hearts_and_Minds.Altis/core/fnc/int/add_actions.sqf
#	=BTC=co@30_Hearts_and_Minds.Altis/core/fnc/int/orders.sqf
#	=BTC=co@30_Hearts_and_Minds.Altis/core/fnc/int/orders_give.sqf

1kuemmel1 committed:
Merge branch 'master_daily' into Remove-double_civ_actions

1kuemmel1 committed:
Merge branch 'master_daily' into Remove-double_civ_actions

Vdauphin committed:
Merge pull request #439 from 1kuemmel1/Remove-double_civ_actions

Add: Remove double actions (orders to civilians)

Vdauphin committed:
Merge pull request #423 from Vdauphin/Add-rep_civ_car

Add: Fire to civilian car decrease reputation

1kuemmel1 committed:
overhaul 1

some changings to place.sqf and place_key_down.sqf

1kuemmel1 committed:
added stringtable entry

1kuemmel1 committed:
added comments

Vdauphin committed:
FIX: detect and find position outside rock

Vdauphin committed:
FIX: debug for findposoutsiderock

Vdauphin committed:
Find rock more accurately

1kuemmel1 committed:
better comments

1kuemmel1 committed:
removed brackets, obj. saving to DB

1kuemmel1 committed:
changed to PosWorld, added comments

1kuemmel1 committed:
added vector handling, used posWorld

Vdauphin committed:
FIX: _rposASL

Vdauphin committed:
Merge pull request #427 from Vdauphin/ADD-change_task

Add: Task description is now localize client side

1kuemmel1 committed:
update mission version

Vdauphin committed:
Remove objNull in btc_vehicles

Vdauphin committed:
Add a waypoint to diver to the closer location

Vdauphin committed:
FIX: remove ACE module

Since ACE 3.12, ACE module are deprecated
(https://slack-files.com/T047TJFFL-F8CF8TVJR-e953dbf392)

Vdauphin committed:
Remove ACE setting from param and use mission.sqm fro them

SInce ACE 3.12, ACE setting is suppose to be choose in mission.sqm or on
the fly by server admin or client. So mission parameter is no more
usefull to change ACE setting. Also add debugconsole for admin ONLY

Vdauphin committed:
Add: advanced engineer to repair vehicles at logistic point

Vdauphin committed:
Merge pull request #451 from Vdauphin/FIX-DB_objnull

FIX: objNull in btc_vehicles

Vdauphin committed:
Merge pull request #453 from Vdauphin/FIX-diver_waypoint

FIX: Diver don't have waypoint on ground

Vdauphin committed:
Merge pull request #454 from Vdauphin/FIX-deprecated

FIX: Remove deprecated ACE module/mission parameters

Vdauphin committed:
Merge pull request #447 from Vdauphin/FIX-spawn_inside_rock

FIX: Enemy spawn inside rock

Vdauphin committed:
FIX: JIP interaction with terminal

Vdauphin committed:
Merge pull request #455 from Vdauphin/FIX-Hack

FIX: Hack side mission

Vdauphin committed:
Add: 1.17.6

Vdauphin committed:
Merge remote-tracking branch 'refs/remotes/origin/master_stable' into master_daily

# Conflicts:
#	=BTC=co@30_Hearts_and_Minds.Altis/core/def/mission.sqf
#	=BTC=co@30_Hearts_and_Minds.Altis/core/def/param.hpp
#	=BTC=co@30_Hearts_and_Minds.Altis/core/doc.sqf
#	=BTC=co@30_Hearts_and_Minds.Altis/core/fnc/compile.sqf
#	=BTC=co@30_Hearts_and_Minds.Altis/core/fnc/mil/create_group.sqf
#	=BTC=co@30_Hearts_and_Minds.Altis/description.ext
#	=BTC=co@30_Hearts_and_Minds.Altis/mission.sqm
#	=BTC=co@30_Hearts_and_Minds.Altis/mission_Tanoa.sqm

Some string are no more need
(https://github.com/Vdauphin/HeartsAndMinds/issues/400)

Vdauphin committed:
Clean up stringtables

Vdauphin committed:
Remove everybody can disarm IED option in mission parameter

Vdauphin committed:
Merge pull request #456 from Vdauphin/FIX-remove_everybody_disarm

FIX: Remove everybody can disarm IED option in mission parameters

Vdauphin committed:
Add: 1.17.7

Vdauphin committed:
Merge remote-tracking branch 'refs/remotes/origin/master_stable' into master_daily

# Conflicts:
#	=BTC=co@30_Hearts_and_Minds.Altis/core/def/mission.sqf
#	=BTC=co@30_Hearts_and_Minds.Altis/core/def/param.hpp
#	=BTC=co@30_Hearts_and_Minds.Altis/core/doc.sqf

Vdauphin committed:
Merge remote-tracking branch 'refs/remotes/origin/master_daily' into Add-clean_up_stringtables

Vdauphin committed:
Add: use IDAP containers for supply

Vdauphin committed:
Add: new icon

Vdauphin committed:
Remove RHS dependency

- mission.sqm : CBA, ACE
- mission_RHSUS_*_sqm : CBA, ACE, RHSUS

Vdauphin committed:
FIX: Arma 3 EDEN editor removed language translation ($STR_BTC_HAM...)

https://github.com/Vdauphin/HeartsAndMinds/issues/400

Vdauphin committed:
Merge pull request #457 from Vdauphin/Add-clean_up_stringtables

Add: Clean up stringtables

Vdauphin committed:
FIX: sometimes engine of vehicles can't be turn ON

Vdauphin committed:
FIX: when the mission fail, markers are not deleted

Vdauphin committed:
trigger is now deleted by btc_fnc_delete

Vdauphin committed:
Merge pull request #462 from Vdauphin/FIX-side_convoy

FIX: Markers are not deleted when side mission convoy/capture officer failed

Vdauphin committed:
Data base also save camp channel

Vdauphin committed:
Merge pull request #463 from Vdauphin/FIX-doc

FIX: Database also save camp channel

Vdauphin committed:
Add: composition to side mission mines

Vdauphin committed:
Merge remote-tracking branch 'refs/remotes/Vdauphin/master_daily' into ADD-overhaul_place_options

# Conflicts:
#	=BTC=co@30_Hearts_and_Minds.Altis/core/def/mission.sqf

Vdauphin committed:
Merge pull request #448 from 1kuemmel1/ADD-overhaul_place_options

Add: Expand place options

Vdauphin committed:
Merge remote-tracking branch 'refs/remotes/origin/master_daily' into FIX-has_suicider_not_save

# Conflicts:
#	=BTC=co@30_Hearts_and_Minds.Altis/core/def/mission.sqf

Vdauphin committed:
Merge pull request #415 from Vdauphin/FIX-has_suicider_not_save

FIX: Suicider spawn after each save and restart

Vdauphin committed:
Merge pull request #459 from Vdauphin/Add-icon

Add: New icons

Vdauphin committed:
Merge pull request #458 from Vdauphin/Add-supply

Add: IDAP containers for supply side mission

bux committed:
Keep CfgFunctions declaration in description.ext

In case someone is editing the mission and wants to add their custom
functions to it.

Vdauphin committed:
No more necessary because those class name are in scope 1

Vdauphin committed:
Reduce conflict between branchs

Vdauphin committed:
space clean up, extrat (), some comments tweaks

Vdauphin committed:
Clean up btc_fnc_log_get_liftable (https://github.com/Vdauphin/HeartsAndMinds/pull/460#issuecomment-355272520)

Vdauphin committed:
Change way to version generate the version

Vdauphin committed:
BTC flag is for global variable only

Vdauphin committed:
Merge pull request #465 from bux/keep-cfgfunctions

Keep CfgFunctions in description.ext

Vdauphin committed:
remove commented _signs

Vdauphin committed:
RMV ()

Vdauphin committed:
Add: random size for the mine field and optimize

Vdauphin committed:
Merge pull request #460 from Vdauphin/Add-ACE_only_sqm

Add: CBA and ACE3 mod only dependency

Vdauphin committed:
migrating to mission.sqf

Vdauphin committed:
Use simple objects for city location

* simple objects
* params
* private

Vdauphin committed:
Merge pull request #466 from Vdauphin/Add-clean_up

Add: Clean up mission.sqf

Vdauphin committed:
FIX: composition follow terrain shape

Vdauphin committed:
Merge remote-tracking branch 'refs/remotes/origin/master_daily' into Add-composition_side_mines

# Conflicts:
#	=BTC=co@30_Hearts_and_Minds.Altis/core/def/mission.sqf

Vdauphin committed:
Add: Land_PlasticNetFence_01_long_d_F

Vdauphin committed:
Merge pull request #464 from Vdauphin/Add-composition_side_mines

Add: Dynamic composition for the side mission clear mines field

Vdauphin committed:
add drone IED

Vdauphin committed:
Add: in database

Vdauphin committed:
FIX: Data base and UAV mouvements

Vdauphin committed:
tweak patrol

Vdauphin committed:
Add first - **Dedicate server support only**

Vdauphin committed:
Tweak

Vdauphin committed:
Rework loops

Vdauphin committed:
Dedicated not Dedicate

Vdauphin committed:
FIX: set damage

create to much damage on unit

Vdauphin committed:
Tweak amount of damage

Vdauphin committed:
FIX: trigger

Vdauphin committed:
Add: optional, 33% of probability and readme

Vdauphin committed:
Add: use ACE3 progressbar

Vdauphin committed:
Clean up revive and interaction

Vdauphin committed:
Clean up English because it is handel by original

Vdauphin committed:
Merge pull request #471 from Vdauphin/Add-rmv_eng

Add: Clean up English from stringtable

Vdauphin committed:
FIX: https://github.com/Vdauphin/HeartsAndMinds/pull/471

Vdauphin committed:
Merge pull request #461 from Vdauphin/FIX-engine_damage

FIX: Sometimes engine of vehicles can't be turn ON

Vdauphin committed:
Merge pull request #469 from Vdauphin/FIX-set_damage

FIX: Set damage fnc for side mission civilian treatment

Vdauphin committed:
rmv useless func

Vdauphin committed:
Merge pull request #467 from Vdauphin/ADD-optimize_by_simple_object

Add: Optimize city creation by using Simple objects

Vdauphin committed:
Merge pull request #470 from Vdauphin/Add-ACE3_progress_bar

Add: Use ACE3 progress bar

Vdauphin committed:
Update issue_template

Vdauphin committed:
FIX: for unknown  reasons ropes are not created for some vehilces

"rhsusf_M1237_M2_usarmy_d" can't be sling loading thanks to
slingloadcargomemorypoints because ropes are not created. the
_ropes_check variable store ropes created and is check after to be sure
ropes are created.

Vdauphin committed:
Add: parameter difficulty for IED spotting

Vdauphin committed:
FIX: factions array due to mods update

Vdauphin committed:
FIX: syntax error from btc_fnc_lift_hud_loop

When `_cargo_y = 0` the   `_hud_y`  was not define :
21:01:19 Error in expression <log str([_hud_x_1, _hud_y_1]);
_obj_img ctrlsetposition [_hud_x_1, _hud_y_1];
_o>
21:01:19   Error position: <ctrlsetposition [_hud_x_1, _hud_y_1];
_o>
21:01:19   Error Type Nombre,Pas un chiffre, Nombre attendu

Vdauphin committed:
Merge pull request #474 from Vdauphin/FIX-faction

FIX: Update factions array due to mods update

Vdauphin committed:
Merge pull request #475 from Vdauphin/FIX-error_lift_hud

FIX: Syntax error from btc_fnc_lift_hud_loop

Vdauphin committed:
Merge pull request #473 from Vdauphin/FIX-slingloading_point

FIX: SlingLoadCargoMemoryPoints during lift

Vdauphin committed:
Change fnc name

Vdauphin committed:
Merge pull request #468 from Vdauphin/ADD-drone_ied

Add: UAV IED

Vdauphin committed:
Merge remote-tracking branch 'refs/remotes/origin/master_stable' into master_daily

# Conflicts:
#	=BTC=co@30_Hearts_and_Minds.Altis/core/def/param.hpp
#	=BTC=co@30_Hearts_and_Minds.Altis/core/doc.sqf

Vdauphin committed:
Add: changelog line in PR template

Vdauphin committed:
Merge remote-tracking branch 'refs/remotes/origin/master_daily' into Add-parameter_IED_spot

# Conflicts:
#	=BTC=co@30_Hearts_and_Minds.Altis/core/def/mission.sqf
#	=BTC=co@30_Hearts_and_Minds.Altis/core/def/param.hpp
#	=BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml

Vdauphin committed:
FIX: missing }

Vdauphin committed:
FIX: when helicopter  wreck are lifted they flip without control

Vdauphin committed:
Merge pull request #476 from Vdauphin/Add-parameter_IED_spot

Add: New mission parameter for IED spotting difficulty

Vdauphin committed:
FIX: lifting wreck helicopter

Vdauphin committed:
Merge remote-tracking branch 'refs/remotes/origin/FIX-lift_wreck_heli' into FIX-lift_flipping

# Conflicts:
#	=BTC=co@30_Hearts_and_Minds.Altis/core/fnc/compile.sqf
#	=BTC=co@30_Hearts_and_Minds.Altis/core/fnc/log/lift_hook.sqf

Vdauphin committed:
FIX: private and missing ;

Vdauphin committed:
Merge pull request #478 from Vdauphin/FIX-lift_wreck_heli

FIX: When helicopter wreck are lifted they flip without control

Vdauphin committed:
Add: ACE Arsenal

Vdauphin committed:
Merge pull request #479 from Vdauphin/Add-Arsenal_ACE

Add: ACE Arsenal

1kuemmel1 committed:
add arsenal handling options

Vdauphin committed:
FIX: suicider no more appear in city if there are killed

Vdauphin committed:
Merge remote-tracking branch 'refs/remotes/origin/master_daily' into FIX-lift_flipping

Use params, private, ()

Vdauphin committed:
Merge pull request #477 from Vdauphin/FIX-lift_flipping

Add: Improve code with `params` and `private` for fnc_lift

Vdauphin committed:
FIX: now customization of vehicles are persistant

Vdauphin committed:
add: Private and params

Vdauphin committed:
FIX: default param

1kuemmel1 committed:
simplify cond. check

1kuemmel1 committed:
fixed AND and UND

Vdauphin committed:
Merge pull request #480 from 1kuemmel1/Add-Arsenal_Selection

Add: Arsenal type parameter

Vdauphin committed:
Merge pull request #481 from Vdauphin/FIX-suicider_spawn

FIX: Now when suicider/drone is killed, the variable `has_suicider` is set to false

Vdauphin committed:
Merge pull request #482 from Vdauphin/FIX-vehicle_customization_DB

FIX: vehicle customization is not persistant

1kuemmel1 committed:
Merge pull request #1 from Vdauphin/master_daily

Update Master daily

Vdauphin committed:
FIX: tweak space

Vdauphin committed:
Merge pull request #483 from Vdauphin/FIXstringtable

FIX: Tweak space

1kuemmel1 committed:
Merge pull request #2 from Vdauphin/master_daily

Update master daily

1kuemmel1 committed:
add function

1kuemmel1 committed:
add mission parameters

- include parameter handling

1kuemmel1 committed:
handle ask_reputation

- adapted files and made minor changes

1kuemmel1 committed:
update comments

1kuemmel1 committed:
use name for hints

1kuemmel1 committed:
handle ask/hideout/tropps

- adapted files and made minor changes

1kuemmel1 committed:
handle orders

- adapted files and made minor changes
- corrected 'taxi'-functionality (open/close map)
- added message for successfully taxi order

1kuemmel1 committed:
add stringtable use in parameter

1kuemmel1 committed:
remove mission parameters

1kuemmel1 committed:
fixed spacing

1kuemmel1 committed:
removed unnecessary brackets

1kuemmel1 committed:
fixed spacing #2

Vdauphin committed:
FIX: the order order from 1 4 2 3 to 1 2 3 4

1kuemmel1 committed:
corrected spelling

1kuemmel1 committed:
switch to if_then_else structure

1kuemmel1 committed:
switch to if_then_else structure

Vdauphin committed:
Merge pull request #484 from 1kuemmel1/Add-Subtitles

Add: Subtitles for conversations

Vdauphin committed:
Merge pull request #485 from Vdauphin/FIX--order-of-on-civilian

FIX: Civilian order are not in the correct order

Vdauphin committed:
Clean up

Vdauphin committed:
FIX : can't fire after placing an object

Vdauphin committed:
Merge pull request #486 from Vdauphin/FIX-place_obj

FIX: Can't fire after placing an object

Vdauphin committed:
Add: 1.17.8

Vdauphin committed:
Merge remote-tracking branch 'refs/remotes/origin/master_stable' into master_daily

# Conflicts:
#	=BTC=co@30_Hearts_and_Minds.Altis/core/def/mission.sqf
#	=BTC=co@30_Hearts_and_Minds.Altis/core/doc.sqf

Vdauphin committed:
use CBA_fnc_waitUntilAndExecute in initplayer

Vdauphin committed:
Clean up

Vdauphin committed:
FIX: remove weapon when cahce is destroyed

Vdauphin committed:
Merge pull request #488 from Vdauphin/FIX-cache

FIX: Remove weapons when cache is destroyed

Vdauphin committed:
Merge pull request #487 from Vdauphin/Add-CBA_initplayer

Add: CBA waitUntilAndExecute in initplayer

bux committed:
change tabs to spaces

might produce some weird looking indentation. Those places should either
be changed in this PR or with every further edit of the respective
files.

bux committed:
add gitattributes file

bux committed:
remove sqm from gitattributes

Vdauphin committed:
Merge pull request #494 from bux/add-editorconfig-fix-tabs

Add editorconfig, fix tabs, normalize line endings

Vdauphin committed:
Merge remote-tracking branch 'refs/remotes/origin/master_stable' into master_daily

# Conflicts:
#	=BTC=co@30_Hearts_and_Minds.Altis/core/fnc/cache/hd_cache.sqf

Vdauphin committed:
Add: 1.18

Vdauphin committed:
Merge remote-tracking branch 'refs/remotes/origin/master_daily' into master_stable

bux committed:
Merge remote-tracking branch 'refs/remotes/Vdauphin/master_stable' into master_stable

1kuemmel1 committed:
Merge pull request #6 from Vdauphin/master_stable

Update master stable

1kuemmel1 committed:
add missing entries

Vdauphin committed:
Merge pull request #499 from 1kuemmel1/FIX-add_missing_translations

FIX: Add missing translations

bux committed:
Merge branch 'master_stable' of https://github.com/Vdauphin/HeartsAndMinds into master_stable

bux committed:
fix two missing functions when placing civs as zeus

Vdauphin committed:
Merge pull request #501 from bux/fix-zeus-funcs

Fix missing functions on client

Vdauphin committed:
Add to  player locality other fnc from common folder

Vdauphin committed:
FIX: veh customiazation for repair wrecks and respawn fnc

Vdauphin committed:
Merge pull request #502 from Vdauphin/FIX-locality

FIX: Missing functions on client

Vdauphin committed:
Merge pull request #503 from Vdauphin/FIX-veh_customization

FIX: Vehicle customization for repair wrecks and respawn fonction

Vdauphin committed:
FIX: side mission JIP

Vdauphin committed:
Merge pull request #504 from Vdauphin/FIX-JIP-side-mission

FIX: Side mission JIP

Vdauphin committed:
FIX: smoke grenade trigger firenear EH

https://github.com/Vdauphin/HeartsAndMinds/issues/512

Vdauphin committed:
Merge pull request #513 from Vdauphin/FIX-smoke_firenear

FIX: Smoke grenade trigger firenear EH

Vdauphin committed:
Add guilines from ACE3 and adjustements

Vdauphin committed:
Add `.`

Vdauphin committed:
Add .

Vdauphin committed:
Add: 1.18.1

Vdauphin committed:
Merge pull request #515 from Vdauphin/Add-guideline

FIX: guidelines missing in PR template

Vdauphin committed:
FIX: BTC logo

Vdauphin committed:
FIX: debug marker have wrong color after restart for saved game

Vdauphin committed:
Merge pull request #534 from Vdauphin/FIX-debug_marker

FIX: Debug marker have wrong color after restart for saved game

Vdauphin committed:
FIX:fleeing animation by removing transition

Vdauphin committed:
Merge pull request #541 from Vdauphin/FIX-animation_fleeing

FIX: Animation fleeing

Vdauphin committed:
update mod support

Vdauphin committed:
Merge pull request #549 from Vdauphin/update-mod-support

FIX: Mod support

Vdauphin committed:
FIX: remove patrol on foot

Vdauphin committed:
Merge pull request #553 from Vdauphin/FIX-foot_patrol_not_removed

FIX: Remove patrol on foot

Vdauphin committed:
FIX: can't drop leaflets

Vdauphin committed:
Re selectweaponturret at each drone unpack

Vdauphin committed:
Update 1.18.2

Vdauphin committed:
FIX: disable AI self healing

Vdauphin committed:
Add:CFP support

https://github.com/tupolov/cfp/releases

Vdauphin committed:
FIX: increase time even more

Vdauphin committed:
Merge pull request #560 from Vdauphin/FIX-side_treatment

FIX: AI is healing him self in side mission treatment

Vdauphin committed:
FIX: missing spanish translation

Vdauphin committed:
FIX: spanish to originzl

Vdauphin committed:
FIX: weapon holdersimulated isn't clean up

Vdauphin committed:
Check point spawn closer to the city center so enemies are around

Vdauphin committed:
Lower even more

Vdauphin committed:
Merge pull request #566 from Vdauphin/FIX-clean_up_weaponholderSimulated

FIX: weapon holdersimulated isn't clean up

Vdauphin committed:
Merge pull request #565 from Vdauphin/FIX-missing_spanish_localization

FIX: Missing spanish localization

Vdauphin committed:
Merge pull request #567 from Vdauphin/FIX-checkpoint

FIX: Checkpoint are too far from the city center

Vdauphin committed:
FIX: when saved hideout are shifted along z

Vdauphin committed:
Merge pull request #573 from Vdauphin/FIX-hideout_shift

FIX: When saved hideout are shifted along z

Vdauphin committed:
Merge pull request #562 from Vdauphin/Add-CFP

FIX: Community Factions Project support

Vdauphin committed:
Add: 1.18.3

Vdauphin committed:
Migrate github wiki to docs
